### PR TITLE
Add Mobius.money to Defi Llama

### PR DESCRIPTION
**Pull request by kyscott18**
##### Twitter Link:
https://twitter.com/MobiusMoney

##### List of audit links if any:
https://github.com/nerve-finance/contracts/blob/main/audits/Certik%20-%20REP-Nerve_Finance_Core_Contracts-06_04_2021.pdf
https://github.com/nerve-finance/contracts/compare/main...mobiusAMM:main

##### Website Link:
https://mobius.money

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://github.com/mobiusAMM/mobius-interface/blob/main/src/assets/images/MOBI-200.png

##### Current TVL:
$32,728,096

##### Chain:
Celo

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
mobius-money

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
N/A

##### Short Description (to be shown on DefiLlama):
A cross-chain stableswap protocol built on Celo.

##### Token address and ticker if any:
0x73a210637f6f6b7005512677ba6b3c96bb4aa44b MOBI

##### Category (Yield/Dexes/Lending/Minting/Assets/Insurance/Options/Indexes/Staking) *Please choose only one:
Dexes

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
None

##### forkedFrom (Does your project originate from another project):
Nerve Finance

### CODE REVIEW

File: projects/mobius/index.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  [pUSDC]: 6,
+  ["celo-dollar"]: 0,
+};
+
+// {[string: name]: {
+//   address: string,
+//   peggedTo: string
+//   tokens: string[],
+// }}
+const pools = {
```
Comment (0xngmi) [inline projects/mobius/index.js:32]: any way we can get these automatically?

File: projects/mobius/index.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+}
+
+/*==================================================
+  Exports
+  ==================================================*/
+
+module.exports = {
+  name: "Mobius", // project name
+  website: "https://mobius.money",
+  category: "DEXes", // allowed values as shown on DefiPulse:
```
Comment (0xngmi) [inline projects/mobius/index.js:139]: we accept any category, you can even make your own

File: projects/mobius/index.js (hunk 1 above)
Comment (kyscott18) [reply projects/mobius/index.js:32]: The way the bridge works, they send the headers like decimals, name, and symbol not with every transaction but with a batch. They haven't been updated yet for all of them so we can't read the correct values from the contracts

Comment (0xngmi) [reply projects/mobius/index.js:32]: got it! we will be relying on the community to keep it updated through PRs then :)

**Pull request merged**